Source/loadsave.cpp

```diff
@@ -1418,32 +1429,33 @@ static void SavePlayer(SaveHelper *file, int p)
 
 	file->skip(2); // Alignment
 
-	file->writeLE<int32_t>(pPlayer->_pGFXLoad);
+	// Write _pGFXLoad for vanilla compatibility
+	file->writeLE<int32_t>(0);
 	file->skip(4 * 8); // Skip pointers _pNAnim
 	file->writeLE<int32_t>(pPlayer->_pNFrames);
-	file->writeLE<int32_t>(pPlayer->_pNWidth);
+	file->writeLE<int32_t>(pPlayer->GetAnimationWidth(player_graphic::Stand));
 	file->skip(4 * 8); // Skip pointers _pWAnim
 	file->writeLE<int32_t>(pPlayer->_pWFrames);
-	file->writeLE<int32_t>(pPlayer->_pWWidth);
+	file->writeLE<int32_t>(pPlayer->GetAnimationWidth(player_graphic::Walk));
 	file->skip(4 * 8); // Skip pointers _pAAnim
 	file->writeLE<int32_t>(pPlayer->_pAFrames);
-	file->writeLE<int32_t>(pPlayer->_pAWidth);
+	file->writeLE<int32_t>(pPlayer->GetAnimationWidth(player_graphic::Attack));
 	file->writeLE<int32_t>(pPlayer->_pAFNum);
 	file->skip(4 * 8); // Skip pointers _pLAnim
 	file->skip(4 * 8); // Skip pointers _pFAnim
 	file->skip(4 * 8); // Skip pointers _pTAnim
 	file->writeLE<int32_t>(pPlayer->_pSFrames);
-	file->writeLE<int32_t>(pPlayer->_pSWidth);
+	file->writeLE<int32_t>(pPlayer->GetAnimationWidth(player_graphic::Magic));

```

Comment:
Do we even need to write these values? I'm thinking the original will set them as it loads the various animations.

Reply:
Checked and thease values are overwritten by LoadGameLevel() right after LoadPlayer() is called so we can just skip them.

Reply:
Thanks for testing 🙂 
I removed the entire `GetAnimationWidth` as the only caller was `LoadPlrGFX`.